### DISCUSSION TITLE
error with Mp3Notify class

### DISCUSSION THREAD

**Discussion by Ingegneus** [Makuna/DFMiniMp3#92] [Q&A]
I am using VS Code and the library version is 1.1.0. I can't get my code to compile since all the
```
T_NOTIFICATION_METHOD::OnPlayFinished(*this, DfMp3_PlaySources_Usb, replyArg);
.
.
.
T_NOTIFICATION_METHOD::OnError(*this, replyArg);
```
throw an error like:
`error: no matching function for call to 'MP3ReplyManager::OnPlayFinished(DFMiniMp3<SoftwareSerial, MP3ReplyManager>&, DfMp3_PlaySources, uint16_t&)`
I am no real coder and C++ is totally new for me, but `*this`
`T_NOTIFICATION_METHOD::OnPlayFinished(→ *this,←  DfMp3_PlaySources_Usb, replyArg);` 
in the deklarations above seem to be the problem, since the functions in the class Mp3Notify only have one argument.

**Comment by Makuna** [comment]
I think you answered you own question, but you don't realize it.  You state "the function in the class Mp3Notify only have one argument".  This Mp3Notify class is implemented by you (it's in your sketch) and it is missing a required argument.  You can fix that in your sketch by adding it.  If all you did was copy the examples, then look at the examples again and you will see they have the extra parameter.

**Reply by Ingegneus** [reply]
Ohhh 🤦‍♂️
I just copied the examples from the Arduino IDE but that version doesn't match the one I am using... Thank you very much. Sorry for the stupid question.
